@here/harp-mapview/lib/MapView.ts

```diff
@@ -983,10 +969,7 @@ export class MapView extends THREE.EventDispatcher {
             mapPassAntialiasSettings
         );
 
-        this.m_tileGeometryManager =
-            this.m_options.enablePhasedLoading === true
-                ? new PhasedTileGeometryManager(this)
-                : new SimpleTileGeometryManager(this);
+        this.m_tileGeometryManager = new SimpleTileGeometryManager(this);
```

Comment:
What is this manager doing? Do we still need it? Maybe rename `SimpleTileGeometryManager` to `TileGeometryManager`